.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
So in that case what you could do is this:

```ini
[testenv:ansible2-lint]
setenv=
    ANSIBLE_VERSION=">=2.0"
deps=
    {[testenv:ansible-lint]deps}
commands=
    {[testenv:ansible-lint]commands}
```

But yes, otherwise that would need to be the invocation to do 2.0 testing